README.md

```diff
@@ -12,22 +12,28 @@
 [![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Fgetindata%2Fkedro-airflow-k8s.svg?type=shield)](https://app.fossa.com/projects/git%2Bgithub.com%2Fgetindata%2Fkedro-airflow-k8s?ref=badge_shield)
 ## About
 
-_**WORK IN PROGRESS**_
-
 The main purpose of this plugin is to enable running kedro pipeline with Airflow on Kubernetes Cluster. In difference to 
 [kedro-airflow](https://github.com/quantumblacklabs/kedro-airflow) this plugin does not require additional libraries installed
-in airflow runtime, it uses K8S infrastructure instead. 
+in airflow runtime, it uses K8S infrastructure instead. It supports translation
+from Kedro pipeline DSL to [airflow](https://airflow.apache.org/docs/apache-airflow/stable/python-api-ref.html) (python API)
+and generation of airflow [DAGs](https://airflow.apache.org/docs/apache-airflow/stable/concepts.html#dags).
 
 The plugin can be used together with `kedro-docker` to simplify preparation of docker image for pipeline execution.   
 
 ## Documentation
 
-**work in progress**
+For detailed documentation refer to https://kedro-airflow-k8s.readthedocs.io/
 
 ## Usage guide
 
-**work in progress** 
+```
+Usage: kedro airflow-k8s [OPTIONS] COMMAND [ARGS]...
+ 
+Options:
+  -e, --env TEXT  Environment to use.
+  -h, --help      Show this message and exit.
 
-## Configuration file
+Commands:
+  generate  Create an Airflow DAG for a project
```

Comment:
Maybe change `generate` to `compile` to match the naming convention with kedro-kubeflow plugin? @szczeles - what do you think?

Reply:
I like the idea!

Reply:
Ok, done.